ezyrb/ann.py

```diff
@@ -25,9 +25,47 @@ class ANN(Approximation):
     >>> print(y_pred)
     """
         
-    def __init__(self):
-        self.trained_epoch = 0   
-        self.criterion = torch.nn.MSELoss()
+    def __init__(self, layers, function, stop_training, loss=None):
+
+        if loss is None: loss = torch.nn.MSELoss
+        if optimizer is None: optimizer = torch.optim.Adam
```

Comment:
"optimizer" has not been introduced yet, I suggest to move this line after line 91.